### ISSUE TITLE
Add item_bonus.inc data to be readily accessible as json data

### ISSUE THREAD

**Issue by Niphyr** [MechanicalPriest/SimcProfileParser#14]
See #10 for information on previous implementations.

Another file needs to be added from this file: https://github.com/simulationcraft/simc/blob/shadowlands/engine/dbc/generated/item_bonus.inc

Structure is an array of `item_bonus_entry_t` named `__item_bonus_data `:

```c++
static const std::array<item_bonus_entry_t, 9813> __item_bonus_data { {
```
`item_bonus_entry_t` structure:
```c++
struct item_bonus_entry_t
{
  unsigned id;
  unsigned bonus_id;
  unsigned type;
  int      value_1;
  int      value_2;
  int      value_3;
  int      value_4;
  unsigned index;
}
```
Previously built class:
```csharp
    internal class SimcRawItemBonusEntry
    {
        public uint Id { get; set; }
        public uint BonusId { get; set; }
        public uint Type { get; set; }
        public int Value1 { get; set; }
        public int Value2 { get; set; }
        public int Value3 { get; set; }
        public int Value4 { get; set; }
        public uint Index { get; set; }
    }
```

**Comment by Niphyr** [comment]
Added as part of https://github.com/MechanicalPriest/SimcProfileParser/commit/de9f1b7a68e67bff4a61f46870ac34028fbf03aa

**Comment by Niphyr** [comment]
Closed with #22
